packages/browser/src/integrations/trycatch.ts

```diff
@@ -176,9 +176,11 @@ export class TryCatch implements Integration {
       ): () => void {
         let callback = (fn as any) as WrappedFunction;
         try {
-          callback = callback && (callback.__sentry_wrapped__ || callback);
+          if (callback && callback.__sentry__) {
+            original.call(this, eventName, callback.__sentry_original__, options);
```

Comment:
Ah so we have to call remove event listener twice. Makes sense.

Also I would appreciate a comment on why this is happening.